### ISSUE TITLE
Results on Squad v2

### ISSUE THREAD

**Issue by adymaharana** [ecchochan/roberta-tpu#1]
Hi, 

Thanks for sharing your code for fine-tuning Roberta on Squad, especially considering that the authors haven't shared the code yet. Could you add some info on the results you are getting using this setup? I read in a related issue opened in the fairseq repo, that you are getting 85 F-Score instead of the 89 reported in the paper. Have you had in the improvements in the results since then? Let me know :) Thanks a ton !

**Comment by adymaharana** [comment]
Also, do you mind sharing the trained weights of your model? Thanks!

**Comment by ecchochan** [comment]
Now the highest F1 is `89.6` (dev set), as compared to `89.4` (dev set) and `89.8` (test set) in the leader board.

Note that the original roBERTa paper achieves `89.4` (dev set) **without decayed learning rate**, as they states **"we use the same learning rate for all layers."**

I will wait until the original author releases the code so I can find out what I have missed! :D

Link to the checkpoint in readme.md.

**Comment by cformosa** [comment]
Really appreciate your detailed work on this. It's very helpful!

**Comment by adymaharana** [comment]
Thanks for the very detailed information on your experiments and for sharing the weights too. This is very helpful! :)
